apps/web/modules/ui/components/dialog/index.test.tsx

```
@@ -0,0 +1,218 @@
+  test("DialogDescription renders correctly", () => {
+    render(<DialogDescription className="test-class">Dialog Description</DialogDescription>);
+
+    const description = screen.getByTestId("dialog-description");
+    expect(description).toBeInTheDocument();
+    expect(description).toHaveClass("test-class");
+    expect(screen.getByText("Dialog Description")).toBeInTheDocument();
+  });
+
+  test("DialogHeader handles dangerouslySetInnerHTML", () => {
+    const htmlContent = "<span>Dangerous HTML</span>";
+    render(<DialogHeader dangerouslySetInnerHTML={{ __html: htmlContent }} />);
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)